expertsystem/io/_pdg.py

```diff
@@ -0,0 +1,129 @@
+"""Create a `.ParticleCollection` instance from PDG info."""
+
+from typing import Optional, Tuple
+
+from particle import Particle as PdgDatabase
+from particle.particle import enums
+
+from expertsystem.data import (
+    Parity,
+    Particle,
+    ParticleCollection,
+    QuantumState,
+    Spin,
+)
+
+
+# cspell:ignore pdgid
+def __get_lepton_qn(pdg_particle: PdgDatabase) -> tuple:
+    electron_lepton_number = 0
+    muon_lepton_number = 0
+    tau_lepton_number = 0
+    if pdg_particle.pdgid.is_lepton:
+        if "e" in pdg_particle.name:
+            electron_lepton_number = pdg_particle.pdgid / abs(
```

Comment:
Nice trick :) But better to compute it once, right after the `if pdg_particle.pdgid.is_lepton`